input/pagecontent/comparison.md

```diff
@@ -1,1206 +1,499 @@
+            <td style="width: 25%;"><a href="https://hl7.org/fhir/us/core/STU7/StructureDefinition-us-core-organization.html">US Core Organization</a></td>
+            <td style="width: 25%;">Organization.active</td>
+            <td style="width: 25%;">US Core requires minimum of 1.</td>
+        </tr>
+        <tr>
+            <td rowspan="3" style="width: 25%;"><a href="https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-patient.html">AU Core Patient</a></td>
+            <td rowspan="3" style="width: 25%;"><a href="https://hl7.org/fhir/us/core/STU7/StructureDefinition-us-core-patient.html">US Core Patient</a></td>
+            <td style="width: 25%;">Patient.identifier</td>
+            <td style="width: 25%;">US Core requires all identifiers to be valid (have system and value).</td>
+        </tr>
+        <tr>
+            <td style="width: 25%;">Gender identity</td>
+            <td style="width: 25%;">US Core defines an US Core Gender Identity extension. AU Core uses the International Gender Identity extension.</td>
+        </tr>
+        <tr>
+            <td style="width: 25%;">Patient.telecom</td>
+            <td style="width: 25%;">US Core requires minimum of 1 system and value.</td>
+        </tr>
+        <tr>
+            <td style="width: 25%;"><a href="https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-practitioner.html">AU Core Practitioner</a></td>
+            <td style="width: 25%;"><a href="https://hl7.org/fhir/us/core/STU7/StructureDefinition-us-core-practitioner.html">US Core Practitioner</a></td>
+            <td style="width: 25%;">Practitioner.identifier</td>
+             <td style="width: 25%;">US Core requires minimum of 1, and requires all practitioner identifiers to have system and value.</td>
+        </tr>
+        <tr>
+            <td style="width: 25%;"><a href="https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-practitionerrole.html">AU Core PractitionerRole</a></td>
+            <td style="width: 25%;"><a href="https://hl7.org/fhir/us/core/STU7/StructureDefinition-us-core-practitionerrole.html">US Core PractitionerRole</a></td>
+            <td style="width: 25%;">PractitionerRole.telecom, PractitionerRole.endpoint</td>
+            <td style="width: 25%;">US Core has constraint pd-1, which mandates that resources SHALL have contact information or a reference to an Endpoint.</td>
+        </tr>
+        <tr>
+            <td rowspan="3" rowspan="3" style="width: 25%;"><a href="StructureDefinition-au-core-smokingstatus.html">AU Core Smoking Status</a></td>
+            <td rowspan="3" style="width: 25%;"><a href="https://hl7.org/fhir/us/core/STU7/StructureDefinition-us-core-smokingstatus.html">US Core Smoking Status Observation Profile</a></td>
+            <td style="width: 25%;">Observation.status</td>
+            <td style="width: 25%;">US Core allows 'final' or 'entered-in-error'.</td>
```

Comment:
Rephrase "US Core requires status of 'final' or 'entered-in-error'

Reply:
Done